source/util/FileUtils.cpp

```diff
@@ -66,7 +66,13 @@ string FileUtils::ExtractExpandedPath(const string &filePath)
         return "";
     }
     wordexp_t word;
-    wordexp(filePath.c_str(), &word, 0); // TODO: How to handle error?
+    int status = wordexp(filePath.c_str(), &word, 0); // TODO: How to handle error?
+    if (status)
+    {
+        throw Aws::Iot::DeviceClient::Util::wordexp_fail_error(
+            "Recieved status from wordexp: " + std::to_string(status) +
```

Comment:
Received is misspelled and I don't think you'd want a new line here.

Reply:
Fixed typo! The new line is to keep the line was added by linter

Reply:
I meant the "\n" in the output. No reason for that I think.

Reply:
resolved